### PR TITLE
[FLOC-1298] Don't delete maintence branches until everything is merged.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1298

### CODE REVIEW

File: docs/gettinginvolved/contributing.rst [hunk 1]
```diff
@@ -43,6 +43,9 @@ The following is the procedure for fixing them.
 
    https://github.com/ClusterHQ/flocker/compare/release/flocker-1.2.3...release-maintenance/flocker-1.2.3/fix-a-bug-FLOC-1234?expand=1
 
+   Note in the pull request, that the branch shouldn't be deleted until every affect release and master have received the fix.
```
Comment (exarkun) [inline docs/gettinginvolved/contributing.rst]: tooootally spurious comma

Comment (exarkun) [inline docs/gettinginvolved/contributing.rst]: s/affect/affected/

**Comment by exarkun** [comment]
I'm not sure how deleting the branch matters.  If someone starts forward-porting the work and then someone adds more commits to the original branch, those commits will be "lost" too won't they?

**Comment by tomprince** [comment]
Well, I tried to get those commits by doing `git merge origin/release-maintenance/flocker-0.3.2/branch-name`. Which succeeded, but didn't have the commits that landed between when I last pulled and when the branch was merged and deleted.

**Comment by tomprince** [comment]
Also, the instructions say to branch off the original branch, which is tricky if it has been deleted.

**Comment by exarkun** [comment]
Okay.  That makes sense.  Maybe we can make this even safer later (prohibiting new development on already-merged branches?  Dunno) but this is definitely good instruction for now.  Thanks, please merge.

**Pull request merged**